### PR TITLE
Tunde

### PR DESCRIPTION
**Pull request by whoistunde**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+        "hostname": "${hostName}",
+        "platform": "${platform}",
+        "architecture": "${architecture}",
+        "numberOfCPUS": "${numberOfCPUS.length}",
+        "networkInterfaces": "${networkInterfaces}",
+        "uptime" : "${uptime}"
+        }`;
+
+        // parse json
+        const jsonObj = JSON.parse(jsonData);
```
Comment (kelue) [inline server.js:48]: Is there really a need to parse this here at this point?

File: package.json [hunk 2]
```diff
@@ -26,5 +27,8 @@
   "bugs": {
     "url": "https://github.com/kelue/SHCapstoneOne/issues"
   },
-  "homepage": "https://github.com/kelue/SHCapstoneOne#readme"
+  "homepage": "https://github.com/kelue/SHCapstoneOne#readme",
+  "dependencies": {
+    "nodemon": "^2.0.15"
```
Comment (kelue) [inline package.json:32]: nodemon should be installed as a dev dependency

**Review by kelue — CHANGES_REQUESTED** [review]

File: server.js [hunk 3]
```diff
@@ -1,2 +1,75 @@
...
+                console.log("An error occured while writing JSON Object to File.");
+                return console.log(err);
+            }
+
+            console.log("JSON file has been saved.");
+            res.end('Your OS info has been saved successfully!');
+        });
+    } 
+    else {
+        res.statusCode = 400;
```
Comment (kelue) [inline server.js:66]: status code here should be 404 not 400

Comment (ahmadinjo) [reply server.js:66]: Plus he should show the 404.html page, right?

**Pull request merged**